### DISCUSSION TITLE
I have a problem that I can't solve. When assembling the Dataloader it keeps giving k errors

### DISCUSSION THREAD

**Discussion by Struggle-Forever** [pyg-team/pytorch_geometric#4950] [General]
![image](https://user-images.githubusercontent.com/29114869/178106082-73b12acc-65d6-4761-ade5-05296b57862e.png)
`DataListLoader(data_list, batch_size=self.args.batch_size, shuffle=True)`

![image](https://user-images.githubusercontent.com/29114869/178106165-13bf5955-1d6c-4815-aa61-e6b1e141951c.png)

I checked my data and it should be fine.

**Comment by Padarn** [comment]
Hey do you mind providing a little more detail? Its a bit hard to tell what you're doing from the screenshots.

Maybe a simple debugging step would be to try and find out what `key` is causing problems in the collate function, it might give you some hints.

**Reply by Struggle-Forever** [reply]
Thank you! I didn't convert “v” to tensor causing the problem.

**Reply by Padarn** [reply]
Welcome!
